src/hardware/gus.cpp

```diff
@@ -937,10 +957,10 @@ void Gus::PrintStats()
 	}
 
 	// Calculate and print info about the volume
-	const auto mixer_scalar = std::max(audio_channel->volmain[0],
-	                                   audio_channel->volmain[1]);
+	const auto scalar = std::max(mixer_level->left,
+	                             mixer_level->right);
```

Comment:
Nitpick: hey, it fits in one line now ;)

Reply:
Got it in the migration to the stand-alone class:
`	const auto peak_sample = std::max(peak.left, peak.right);`